samples/unit-tests/axis/bar-reversed-yaxis/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test('Reversed yAxis for bar charts cause detached columns from axis.', function (assert) {
+QUnit.test('Reversed yAxis for bar charts cause detached columns from axis (#4504)', function (assert) {
     var options = {
```

Comment:
This test fails because the elements `container_1`, `container_2` and `container_3` are not present in the karma template. Must refactor the test to either dynamically create elements within `container`, or run charts and assertions one by one.